helm/resource_release.go

```diff
@@ -1163,6 +1167,16 @@ func getChart(d resourceGetter, m *Meta, name string, cpo *action.ChartPathOptio
 	m.Lock()
 	defer m.Unlock()
 
+	// Checks if chart is a URL; checks if it's a valid URL to a .tgz file of the chart
```

Comment:
I don't think this check is necessary – the LocateChart() call below will produce a similar error if you specify a URL to an invalid chart anyway.

Reply:
I would prefer to just defer to the helm module on what error to produce if you give it a URL that isn't a helm chart. The logic below isn't very precise. For example, I can still specify a URL that will return a different kind of binary data, or an archive that isn't a helm chart at all. It also generates an additional HTTP call that has to happen at plan time.

Generating a better error message when a user specifies an invalid URL is something we can figure out in a separate PR – I don't think this is needed to solve the original `.version` issue.